tracing/src/main/java/com/palantir/remoting2/tracing/Tracers.java

```diff
@@ -21,9 +21,13 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ThreadLocalRandom;
+import org.slf4j.MDC;
 
 /** Utility methods for making {@link ExecutorService} and {@link Runnable} instances tracing-aware. */
 public final class Tracers {
+    /** The key under which trace ids are inserted into SLF4J {@link org.slf4j.MDC MDCs}. */
+    static final String MDC_KEY = "traceId";
```

Comment:
I copied this from `Tracers.MDC_KEY` -- not sure where to put it to reduce duplication.